src/robusta/cli/integrations_cmd.py

```diff
@@ -15,22 +16,26 @@
     "SLACK_INTEGRATION_SERVICE_ADDRESS",
     "https://api.robusta.dev/integrations/slack/get-token",
 )
+SlackApiKey = namedtuple('SlackApiKey', 'key team_name')
 
 
-def wait_for_slack_api_key(id: str) -> str:
+def wait_for_slack_api_key(id: str) -> SlackApiKey:
     while True:
         try:
             response_json = requests.get(
                 f"{SLACK_INTEGRATION_SERVICE_ADDRESS}?id={id}"
             ).json()
             if response_json["token"]:
-                return str(response_json["token"])
+                team_name = None
+                if 'team-name' in response_json:
+                    team_name = str(response_json['team-name'])
+                return SlackApiKey(str(response_json["token"]), team_name)
```

Comment:
Can we simplify to return SlackApiKey(str(response_json["token"]), response_json.get("team-name", None)) ?